Codecademy/gamut
# feat(EditableIcon): Add ProgrammingTeamChatIcon

### Overview



Adds ProgrammingTeamChatIcon.



<img width="260" alt="Screenshot 2024-08-06 at 8 50 20 AM" src="https://github.com/user-attachments/assets/84a511a3-5fc0-4ac9-8efb-9414c99e454a">

### PR Checklist

- [x] Related to designs: https://www.figma.com/design/BmF7Aam6DpdGdE4gYRNhOQ/Leaderboard?node-id=130-4759&m=dev
- [x] Related to JIRA ticket: [LEGO-4022](https://skillsoftdev.atlassian.net/browse/LEGO-4022)

#### Testing Instructions

Check out the icon in the [storybook preview](https://66b14ed8bef9a922daae48ec--gamut-preview.netlify.app/?path=/docs/atoms-icons-regular--editable-icon), and in the portal-app environment using the [instructions in the mono PR](https://github.com/codecademy-engineering/mono/pull/7234).

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/7232)  | [Portal Env](https://tengu.codecademy.com/learn?PR_ENV=portal-app-pr-7234)   |




[LEGO-4022]: https://skillsoftdev.atlassian.net/browse/LEGO-4022?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
